adb_graphics/datahandler/gribdata.py

```diff
@@ -112,6 +112,17 @@ def field_diff(self, values, variable2, level2, **kwargs):
 
         return diff
 
+    def field_column_max(self, values, variable, **kwargs):
+
+        # pylint: disable=unused-argument
+
+        ''' Returns the column max of the values. '''
+
+        vals = self.values(name=variable, level='1000ft', one_lev=False)
```

Comment:
I'm curious if the level arg here is necessary. I was thinking that if it's not set, it might be useful for variables that don't have a 1000ft level set.